azure-kusto-data/src/client.rs

```diff
@@ -97,9 +102,25 @@ impl KustoClient {
             pipeline: pipeline.into(),
             query_url: query_url.into(),
             management_url: management_url.into(),
+            default_headers,
         })
     }
 
+    pub(crate) fn default_headers(details: ClientDetails) -> Headers {
+        let mut headers = Headers::new();
+        const API_VERSION: &str = "2019-02-13";
+        headers.insert("x-ms-kusto-api-version", API_VERSION);
+        headers.insert("x-ms-app", details.application);
+        headers.insert("x-ms-user", details.user);
+        headers.add(Accept::from("application/json"));
+        headers.add(ContentType::new("application/json; charset=utf-8"));
+        headers.add(AcceptEncoding::from("gzip"));
+        headers.add(ClientVersion::from(details.version));
+        headers.insert("connection", "Keep-Alive");
```

Comment:
👍 - our previous headers were raising with the latest azure crates due to us capitalising "Connection", great that this is fixed. Haven't looked into it more deeply, but maybe this may happen for user assigned headers as well? Come to think of it, the error message was quite clear, so if it happens users should know what to do :).

Reply:
I agree.
It is a weird stipulation they added though.